source/monthly-calendar.js

```diff
@@ -141,6 +142,56 @@ function checkTasks(ulElt, day, month, year) {
     });
 }
 
+/**
+ * Event handler for hovering over a day to highlight it.
+ * 
+ * @name dayHoverHandler
+ * @function
+ * @param {HTMLElement} liElt - The list item element representing the date.
+ */
+function dayHoverHandler(liElt) {
+    liElt.addEventListener('mouseenter', function() {
+        liElt.classList.add('day-hover');
+    });
+
+    liElt.addEventListener('mouseleave', function() {
+        liElt.classList.remove('day-hover');
+    });
+}
+
+/**
+ * Event handler for clicking on a day to add a task.
+ * 
+ * @name dayClickHandler
+ * @function
+ * @param {HTMLElement} liElt - The list item element representing the date.
+ * @param {number} day - The day of the month.
+ * @param {number} month - The month (0-11).
+ * @param {number} year - The full year.
+ */
+function dayClickHandler(liElt, day, month, year) {
+    liElt.addEventListener('click', function() {
+        const taskTitle = prompt("Enter task title:");
+        if (taskTitle) {
```

Comment:
I'd suggest utilizing functions in task-crud for crud operations since views are updated automatically on change through change listeners in task-crud